data/json/terrain.json

```diff
@@ -3229,6 +3230,7 @@
             "items": [
                  { "item": "2x4", "amount": 3, "minamount": 0 },
                  { "item": "nail", "amount": 8, "minamount": 3 },
+                 { "item": "material_sand", "amount": 1200, "minamount": 800 },
```

Comment:
Might divide by 2 here as smashing seems more likely to get the sand into a mess rather than in a retrievable pile.